src/pages/products/sgid/base-maps/hybrid.astro

```diff
@@ -78,12 +78,12 @@ const page: IPageMetadata = {
 
   <Section title="More resources" slot="more-resources" titlePosition="center">
     <GridForMoreResources>
+      <CardWithPopularLink title="Base Map Maintenance" href="/documentation/discover/maintenance-schedule/"
```

Comment:
Will you lower case map and maintenance on all these please

Reply:
done